### PR TITLE
Feature: UNET model utils

### PR DESCRIPTION
**Pull request by hlfshell**
This PR introduces model utils - specifically it introduces the following functionality:

- A function that accepts a model and an image and returns labels
- A conversion function for a given image into an input tensor for the network
- A conversion function that takes model output and converts it to an image of labels
- An overlay function that combines an input image and a set of labels for an overlayed labeled image.

### CODE REVIEW

File: unet/utils.py [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+import numpy as np
+
+# from keras.preprocessing.image import array_to_img
+from math import floor
+from PIL import Image, ImageOps
+from skimage.transform import resize
+from typing import Optional, Tuple
+
+
+def infer(model, img : Image) -> np.ndarray:
```
Comment (zekeflaton) [inline unet/utils.py:11]: thoughts on my opening a PR to extend this where we create a class that initializes with a specific model, then we can call a class function to process an image?  should let us process multiple images more efficiently using the same model.  Do we think that would be helpful for us?

**Review by zekeflaton — APPROVED** [review]
looks great

File: unet/utils.py (hunk 1 above)
Comment (hlfshell) [reply unet/utils.py:11]: Sure, you can do that, but I don't think we have time to do another model at the moment.
